explainability-service/manifests/opendatahub/default/trusty-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    app.openshift.io/vcs-url: <<unknown>>
+    app.quarkus.io/commit-id: ea73183a2c1b81bd9afe31005644d374b5bdeb34
```

Comment:
Is this the actual commit id of the repo to be used? Should be update it for every release?

Reply:
Ill defer to @ruivieira  on this. I just copied his yamls :-)